app/views/publisher/_aspect_dropdown.html.haml

```diff
@@ -1,5 +1,5 @@
 .btn-group.aspect_dropdown
-  %button.btn.btn-default.dropdown-toggle{ ! current_user.getting_started? ? {'data-toggle' => 'dropdown'} : {'data-toggle' => 'dropdown', title: popover_with_close_html("2. #{t('shared.public_explain.control_your_audience')}"), 'data-content'=> t('shared.public_explain.visibility_dropdown')} }
+  %button.btn.btn-sm.btn-default.dropdown-toggle{ ! current_user.getting_started? ? {'data-toggle' => 'dropdown'} : {'data-toggle' => 'dropdown', title: popover_with_close_html("2. #{t('shared.public_explain.control_your_audience')}"), 'data-content'=> t('shared.public_explain.visibility_dropdown')} }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.